crates/runtime/src/lib.rs

```diff
@@ -132,7 +132,10 @@ pub fn default_config() -> Result<Config> {
 
 // TODO (Joe): expose the wasmtime wasi context as a capability?
 pub fn default_wasi() -> Result<WasiCtx> {
-    let ctx: WasiCtxBuilder = WasiCtxBuilder::new().inherit_stdio().inherit_args()?;
+    let ctx: WasiCtxBuilder = WasiCtxBuilder::new()
```

Comment:
I made the comment in #342 that we might want to consider allowing the CLI to pass a `WasCtx` to the slight runtime.

Reply:
~as new command-line args or part of the slightfile? I'll merge this for now to accommodate AKS EE, but we can change it later.